php-mode.el

```diff
@@ -11,7 +11,7 @@
 (defconst php-mode-version-number "1.15.3"
   "PHP Mode version number.")
 
-(defconst php-mode-modified "2015-01-31"
+(defconst php-mode-modified "2015-02-02"
```

Comment:
Perhaps I am misreading the readme. I will revert it if you prefer. The test is there to prove that #200 should be closed, that's it.